projects/RabbitMQ.Client/client/impl/ChannelBase.cs

```diff
@@ -453,7 +453,7 @@ await c.HandleCommandAsync(cmd)
         {
             if (!_flowControlBlock.IsSet)
             {
-                _flowControlBlock.Wait();
+                _flowControlBlock.Wait(cancellationToken);
```

Comment:
I wonder though if as a subsequent PR it would make sense to add here an AsyncManualResetEvent to not block the callers during flow control

Thoughts @stebet ?

Reply:
@lukebakken any thoughts on this?

Reply:
Makes sense to me, I can add this to 7.1